mapbox-store-locator/ThemeViewController.swift

```diff
@@ -179,13 +182,11 @@ class ThemeViewController: UIViewController, MGLMapViewDelegate, CLLocationManag
         if let name = feature.attribute(forKey: "name") as? String {
             
             // Change the icon to the selected icon based on the feature name. If multiple items have the same name, choose an attribute that is unique.
-            layer.iconImageName = MGLStyleValue(interpolationMode: .categorical,
-                                                sourceStops: [name: MGLStyleValue<NSString>(rawValue: "purple_selected_burger")],
-                                                attributeName: "name",
-                                                options: [.defaultValue: MGLStyleValue<NSString>(rawValue: "purple_unselected_burger")])
+            layer.iconImageName = NSExpression(format: "MGL_MATCH(name, %@, 'selected_marker', 'unselected_marker')", name)
```

Comment:
There are only two cases, so it’d be more natural to say `TERNARY(name = %@, 'selected_marker', 'unselected_marker')` (or use the `NSExpression(forConditional:true:false:)` initializer).